polaris-react/src/components/Button/Button.scss

```diff
@@ -105,9 +105,8 @@
 
   &:last-child {
     // This compensates for the disclosure icon, which is substantially
-    // inset within the viewbox (and makes it look like there is too much
-    // spacing on the right of the button)
-    margin-right: calc(-1 * (var(--p-space-2)));
+    // inset within the viewbox
+    margin-right: calc(-1 * (var(--p-space-1)) - 2px);
```

Comment:
What is the pixel value here? `-2px`?

Reply:
It's a magic number to account for that extra spacing from the icon

<img width="1224" alt="Screen Shot 2022-06-28 at 3 18 24 PM" src="https://user-images.githubusercontent.com/2091116/176267139-2acbad9b-7ed7-4730-a80b-3d3fa512da98.png">

Reply:
Can we just use `--p-space-05`?

https://github.com/Shopify/polaris/blob/main/polaris-tokens/src/token-groups/spacing.ts#L8-L10

Reply:
Oh, absolutely. That will do it 👍🏻